about_enums.exs

```diff
@@ -53,227 +48,220 @@ defmodule AboutEnums do
     assert Enum.at(list, 5) == __?
   end
 
-  think "at can take a default" do
+  think "It can take a default" do
     list = [:a, :b, :c]
     assert Enum.at(list, 5, :something) == __?
   end
 
-  think "fetch is like at" do
+  think "Fetching is similar to at" do
     list = [:a, :b, :c]
     assert Enum.fetch(list, 0) == __?
   end
 
-  think "fetch tells you if it can't find an element" do
+  think "Fetching tells you if it can't find an element" do
     list = [:a, :b, :c]
     assert Enum.fetch(list, 4) == __?
   end
 
-  think "fetch! will raise an exception if it can't find an element" do
+  think "Fetching will raise an exception if it can't find an element" do
     list = [:a, :b, :c]
     assert_raise __?, fn -> Enum.fetch!(list, 4) end
   end
 
-  think "find the first element which matches" do
+  think "Find the first element that matches" do
     list = [1, 2, 3, 4]
     assert Enum.find(list, fn (x) -> x > 2 end) == __?
   end
 
-  think "what happens when find can't find?" do
+  think "What happens when find can't find?" do
     list = [1, 2, 3, 4, 5]
     assert Enum.find(list, fn (x) -> x > 5 end) == __?
   end
 
-  think "find takes a default" do
+  think "Find takes a default" do
     list = [1, 2, 3]
     assert Enum.find(list, 4, fn (x) -> x > 3 end) == __?
   end
 
-  think "what index is this number at?" do
+  think "What index is this number at?" do
```

Comment:
While you're in here, might as well try and avoid the dangling preposition. `think  "What is the index of this number?"`?